swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -197,6 +197,7 @@ let default_translations = {
       container_ops: {
         addContainer: "Add a new bucket",
         editContainer: "Editing bucket: ",
+        norename: "It is not possible to rename buckets.",
```

Comment:
```suggestion
        norename: "At the moment renaming buckets is not possible.",
```

Reply:
The rephrasing gives the impression that it is a planned feature. I had the impression that it is not. Did I understand wrongly?

Reply:
It is not a planned feature, however I think we will get a request for it, at some point.
We can phrase it differently than what i suggested, however what i wanted to express was that it is not supported, but it is possible as users are able to do copy & delete operations to get a new bucket/object.

Reply:
`We do not support renaming buckets/objects.` could be an alternative.

Reply:
A short explanation could be `It is not possible to rename container/object. However, you can copy it with a new name.`

Reply:
sure, that works